ci_handbook/model-output-standards.qmd

```diff
@@ -0,0 +1,45 @@
+| *lat* or *Y*          | **REQUIRED IF PREDICTING ON GRID**. Latitude (units = `degrees_north`).                                                                                                                                                                                                       |
+| *lon* or *X*          | **REQUIRED IF PREDICTING ON GRID**. Longitude (units = `degrees_east`) is the default spatial coordinate. The alternative use of Y, X for spatial coordinates should conform to the CF convention and requires additional metadata about grids and projections.                                                                                                                                                                                                                                          |
+| *site_id*           | **REQUIRED IF NOT PREDICTING ON A GRID**. For predictions that are not on a spatial grid, use a site dimension that maps to a more detailed geometry (points, polygons, etc.). *site_id* should be used when making predictions that map to a stream network geofabric, for example.                                                                                                                                                                                                              |
+| *family*             | **REQUIRED**. Describes the distribution of the prediction (e.g., `normal` for a normal distribution). For ensemble predictions, use `ensemble`. If the predictions yield a single realization (i.e., there is no associated uncertainty), it is preferred to specify `ensemble` with the ensemble size set to 1. If this dimension remains constant, it is acceptable to define *family* as a variable attribute, provided that the file format supports this (e.g., netCDF). Refer to [appendix S1](https://esajournals.onlinelibrary.wiley.com/action/downloadSupplement?doi=10.1002%2Fecs2.4686&file=ecs24686-sup-0001-AppendixS1.pdf) of [@dietze2023community] for supported _family_ and _parameter_ names.                                                                                   |
+| *parameter*          | **REQUIRED**. For ensemble predictions, specify integers from `1` to `Ne` (where `Ne` represents the total size of the ensemble). For named distributions, indicate the parameter or statistic being specified (e.g., `mean`, `sigma`).                                                                                                                                                                                                                                         |
```

Comment:
change to (e.g., a normal distribution would have  `mean` and `sd` as the values in the parameter column)